hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/fs/ozone/TestRootedOzoneFileSystem.java

```diff
@@ -113,12 +125,12 @@ public void teardown() {
 
   @Test
   public void testOzoneFsServiceLoader() throws IOException {
-    OzoneConfiguration conf = new OzoneConfiguration();
+    OzoneConfiguration cfg = new OzoneConfiguration();
```

Comment:
NIT. 'conf' may be a more consistent choice of naming in Ozone repo.

Reply:
This is due to I put another `conf` in global. So IntelliJ is complaining this `conf` inside shadows the global `conf` in this function.

Anyway, will rename it.

Reply:
Done in 28cedc4db0820b84e8d2af151525d66ec2af944b